### PR TITLE
Replacing Adminer in favor of Adminerevo

### PR DESCRIPTION
**Pull request by gogl92**

### CODE REVIEW

File: 7.4.Dockerfile [hunk 1]
```diff
@@ -34,10 +34,10 @@ RUN \
...
     # && apk add -U rabbitmq-server@testing \
     # && apk add -U rabbitmq-server \
-  # adminer
-  && mkdir -p /var/www/adminer \
-    && curl -sSLo /var/www/adminer/index.php \
-      "https://github.com/vrana/adminer/releases/download/v$ADMINER_VERSION/adminer-$ADMINER_VERSION-en.php" \
+  # adminerevo
+  && mkdir -p /var/www/adminerevo \
+    && curl -sSLo /var/www/adminerevo/index.php \
+      "https://github.com/vrana/adminerevo/releases/download/v$ADMINEREVO_VERSION/adminer-$ADMINEREVO_VERSION.php" \
```
Comment (adhocore) [inline 7.4.Dockerfile:40]: vrana ^^

File: README.md [hunk 2]
```diff
@@ -19,26 +19,26 @@ The download size is just about ~350MB which is tiny considering how much tools
...
-PHP8.1`+`     | >=8.1.27   | 9000
-PHP8.0`+`     | >=8.0.30   | 9000
-PHP7.4`~`     | >=7.4.33   | 9000
-PostgreSQL    | 14.7       | 5432
-~rabbitmq~`^` | 3.8.*      | 5672
-redis         | 7.0.10     | 6379
-~swoole~`"`   | 4.8.9      | -
+Name             | Version    | Port
+-----------------|------------|------
+adminerevo       | 4.8.1      | 80
```
Comment (adhocore) [inline README.md:24]: 4.8.4

File: 8.2.Dockerfile [hunk 3]
```diff
@@ -35,10 +35,10 @@ RUN \
   # rabbitmq
   # && apk add -U rabbitmq-server@testing \
     # && apk add -U rabbitmq-server \
-  # adminer
-  && mkdir -p /var/www/adminer \
-    && curl -sSLo /var/www/adminer/index.php \
-      "https://github.com/vrana/adminer/releases/download/v$ADMINER_VERSION/adminer-$ADMINER_VERSION-en.php" \
+  # evo
+  && mkdir -p /var/www/evo \
+    && curl -sSLo /var/www/evo/index.php \
```
Comment (adhocore) [inline 8.2.Dockerfile:40]: dir name is evo or adminerevo?

File: nginx/conf.d/default.conf [hunk 4]
```diff
@@ -9,8 +9,8 @@ server {
   if (-d "/var/www/html/public") {
     set $docroot "html/public";
   }
-  if ($request_filename ~ /adminer) {
-    set $docroot "adminer";
+  if ($request_filename ~ /adminerevo) {
```
Comment (adhocore) [inline nginx/conf.d/default.conf:12]: should we also keep old url working? like redirecting /adminer to /adminverevo automatically

File: 8.2.Dockerfile (hunk 3 above)
Comment (gogl92) [reply 8.2.Dockerfile:40]: Bad text replacing

File: nginx/conf.d/default.conf (hunk 4 above)
Comment (gogl92) [reply nginx/conf.d/default.conf:12]: that can be use ful, I'll make the change

**Review by adhocore — APPROVED** [review]
thank you

**Pull request merged**